covdefaults.py

```diff
@@ -97,6 +97,9 @@ def _version_pragmas(
             r': \.\.\.(\s*#.*)?$',
             r'^ +\.\.\.$',
             r'-> [\'"]?NoReturn[\'"]?:',
+            r'^\s*assert_never\b',
+            r'^\s*typing\.assert_never\b',
+            r'^\s*typing_extensions\.assert_never\b',
```

Comment:
the current enforced conventions require from imports -- so let's remove these two